src/core/tests/Test_AptitudePackageManager.py

```diff
@@ -445,6 +449,65 @@ def test_package_manager_instance_created_even_when_exception_thrown_in_pro(self
 
         UbuntuProClient.UbuntuProClient.install_or_update_pro = backup_package_manager_ubuntu_pro_client_install_or_update_pro
 
+    def test_get_other_updates_success(self):
+        obj = MockVersionResult()
+        obj.mock_import_uaclient_version_module('version', 'mock_version')
+        updates_obj = MockUpdatesResult()
+        updates_obj.mock_import_uaclient_update_module('updates', 'mock_update_list_with_all_update_types')
+        runtime = RuntimeCompositor(ArgumentComposer().get_composed_arguments(), True, Constants.APT)
+        runtime.set_legacy_test_type('UA_ESM_Required')
+
+        backup_AptitudePackageManager__pro_client_prereq_met = runtime.package_manager._AptitudePackageManager__pro_client_prereq_met
+        runtime.package_manager._AptitudePackageManager__pro_client_prereq_met = True
+
+        packages, versions = runtime.package_manager.get_other_updates()
+        self.assertEqual(1, len(packages))
+
+        runtime.package_manager._AptitudePackageManager__pro_client_prereq_met = backup_AptitudePackageManager__pro_client_prereq_met
+        obj.mock_unimport_uaclient_version_module()
+        updates_obj.mock_unimport_uaclient_update_module()
+
+    def test_get_other_updates_without_pro_success(self):
+        runtime = RuntimeCompositor(ArgumentComposer().get_composed_arguments(), True, Constants.APT)
+        runtime.set_legacy_test_type('UA_ESM_Required')
+        backup_package_manager_get_security_updates = runtime.package_manager.get_security_updates
+        runtime.package_manager.get_security_updates = self.mock_get_security_updates_return_empty_list
+
+        packages, versions = runtime.package_manager.get_other_updates()
+        self.assertEqual(1, len(packages))
+
+        runtime.package_manager.get_security_updates = backup_package_manager_get_security_updates
+
+    def test_set_security_esm_package_status_assessment(self):
+        obj = MockVersionResult()
+        obj.mock_import_uaclient_version_module('version', 'mock_version')
+        updates_obj = MockUpdatesResult()
+        updates_obj.mock_import_uaclient_update_module('updates', 'mock_update_list_with_all_update_types')
+        runtime = RuntimeCompositor(ArgumentComposer().get_composed_arguments(), True, Constants.APT)
+        runtime.set_legacy_test_type('UA_ESM_Required')
+        backup_aptitudepackagemanager__pro_client_prereq_met = runtime.package_manager._AptitudePackageManager__pro_client_prereq_met
+        runtime.package_manager._AptitudePackageManager__pro_client_prereq_met = True
+
+        runtime.patch_assessor.start_assessment()
+        status = ""
+        error_set = False
+        with self.runtime.env_layer.file_system.open(self.runtime.execution_config.status_file_path, 'r') as file_handle:
+            status = json.load(file_handle)
+            self.assertEqual(status[0]["status"]["status"].lower(), Constants.STATUS_SUCCESS.lower())
```

Comment:
Also, assert operation name == Assessment, just to be sure

Reply:
The status file does not have the operation name. So, skipping this for now.

Reply:
@SathishMSFT I just realized, it won't be operation name, but this substatus will have:
status[0]["status"]["name"] = "PatchAssessmentSummary"

Please verify that via assert

Reply:
its actually in "status[0]["status"]["substatus"][0]["name"]". Added the check in the test.